apps/web/components/Embed.tsx

```diff
@@ -1144,8 +1145,12 @@ const EmbedTypeCodeAndPreviewDialogContent = ({
                     />
                   )}
                 </div>
-                <div className={router.query.embedTabName == "embed-preview" ? "block" : "hidden"} />
-                <div className="mt-8 flex flex-row-reverse gap-x-2">
+                <div className={router.query.embedTabName == "embed-preview" ? "mt-2 block" : "hidden"} />
+                <DialogFooter
+                  className="mt-8 flex flex-row-reverse gap-x-2"
+                  showDivider
+                  customDividerClassNames="w-2/3">
```

Comment:
In this specific case we had to implement this to the divider fill only the desired component space:
Without this prop:
![244967944-bda8f054-f3da-4c30-b075-7d279a477396](https://github.com/calcom/cal.com/assets/121884634/098d6786-2b3a-42a9-98c2-f3b4b7babd6b)

Reply:
The divider should not cross into the gray part on the left ideally

Reply:
Ok we did this, the screenshot above is just to show why we need this prop we added 😄